c/driver/sqlite/sqlite_test.cc

```diff
@@ -826,6 +828,32 @@ TEST_F(SqliteReaderTest, InferTypedParams) {
                   "[SQLite] Type mismatch in column 0: expected INT64 but got DOUBLE"));
 }
 
+TEST_F(SqliteReaderTest, BindByName) {
+  adbc_validation::StreamReader reader;
+  Handle<struct ArrowSchema> schema;
+  Handle<struct ArrowArray> batch;
+
+  ASSERT_THAT(adbc_validation::MakeSchema(&schema.value,
+                                          {
+                                              {"@b", NANOARROW_TYPE_INT64},
```

Comment:
I may find this as I keep looking but does the ADBC SQLite driver support [all five parameter syntaxes](https://www.sqlite.org/lang_expr.html#varparam) SQLite itself does? And then it looks like DBAPI 2.0 limits this to just two styles (qmark, named), so then would/should the DBAPI interface only support `?` and `:AAAA`?

Reply:
We technically ignore that part of DBAPI and just pass through whatever the underlying vendor uses. 

The driver should support all the syntaxes now.